supervisor.go

```diff
@@ -257,6 +258,7 @@ func (s *timedSupervisor) Start() error {
 			}
 		}
 	}()
+	s.mutex.Unlock()
```

Comment:
Why after go func() and not just after s.t = ... ?

Reply:
I was thinking that mutex should unlock only after whatever happened in go func(), but as i just checked, it's also works when it's after s.t =...
Fixed.